user/build-environment-updates/2018-06-15.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Build Environment Update History
+layout: en
+permalink: /user/build-environment-updates/2018-06-15/
+category: mac_build_env_updates
+date: 2018-06-15
+---
+
+## 2018-06-15
+
+This update applies to `osx_image: xcode9.4.1`, for osx builds
+Xcode 9.4.1
+Build version 9F2000
+
+### Schedule
+
+2018-06-15 19:00 UTC
+
+This xcode image is running on High Sierra 10.13
```

Comment:
Is this normally part of the schedule, or should this line go in the above section?

Reply:
ah. i think you're right.  the last release was a change to high sierra for the first time.  I'll switcheroo